webpack.common.js

```diff
@@ -13,7 +13,10 @@ const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const MonacoWebpackPlugin = require("monaco-editor-webpack-plugin");
 
 const isPatternflyStyles = (stylesheet) =>
-  stylesheet.includes("@patternfly/react-styles/css/") ||
+  (stylesheet.includes("@patternfly/react-styles/css/") &&
+    !stylesheet.includes(
+      "@patternfly/react-styles/css/components/Topology/"
+    )) ||
```

Comment:
Could you please add a comment explaining why we needed this?
We'll likely forget it.
The issue here was that the styles of the topology component are not included in the patternfly-base stylesheet. We needed to allow them to be imported from the `@patternlfy/react-styles` repo, hence the change.